formfactory/__init__.py

```diff
@@ -1,28 +1,65 @@
 from django.conf import settings
 
 
+class AppFields(object):
+    DJANGO = "django.forms.fields"
+    FORMFACTORY = "formfactory.fields"
+
+class AppWidgets(object):
+    DJANGO = "django.forms.widgets"
+    FORMFACTORY = "formfactory.widgets"
+
 SETTINGS = getattr(settings, "FORMFACTORY", {
```

Comment:
Tbh I'm not a fan of aliasing settings like this because it means override_settings in tests stop working. However, this is not part of the work you've done so ignoring it.